pkg/plugins/layout.go

```diff
@@ -457,7 +478,12 @@ func (dev *Disk) ExpandLastPartition(l logger.Interface, size uint, console Cons
 
 	gd.DeletePartition(part.Number)
 	gd.CreatePartition(&part)
-	out, err := gd.WriteChanges(console)
+	isGPT, err := dev.IsGPT(console)
```

Comment:
cant we check this directly on the WriteChanges?

Reply:
yeah we can ... tbh I'm not very happy with either at the moment, because like you say it feels like something that should be in WriteChanges, but at the same time in that function we don't have access to dev, so I will have to load it, which means things will get more tightly coupled ... will give it another spin and see how it feels

Reply:
ok decided to make the relationship between GdiskCall and Disk closer, I think that's the best compromise in this case, and that way I could bring it into WriteChanges, let me know what you think